Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -3,26 +3,23 @@ import SwiftUI
 import UserNotifications
 
 struct DashboardView: View {
+    @State private var showRandomEventSentAlert = false
+
     @State private var showingCustomEventSheet = false
     @State private var showingCustomEventAlert = false
     @State private var customEventName: String = ""
     @State private var customEventPropertyName: String = ""
     @State private var customEventPropertyValue: String = ""
 
-    @State private var showingDeviceAttributesSheet = false
-    @State private var showingDeviceAttributesAlert = false
-    @State private var deviceAttributeName: String = ""
-    @State private var deviceAttributeValue: String = ""
-
-    @State private var showingProfileAttributesSheet = false
-    @State private var showingProfileAttributesAlert = false
-    @State private var profileAttributeName: String = ""
-    @State private var profileAttributeValue: String = ""
-
     @EnvironmentObject var userManager: UserManager
 
     @State private var showSettings: Bool = false
-    @State private var showAskForPushPermissionButton = false
+
```

Comment:
I'm trying to think about all these booleans. They seem really heavy for what they are accomplishing. Could you not set a "current view" or something and do a logic check instead of tracking individual booleans for everything? The answer might be no, and if it is I probably need to learn a little more in here to show a possible second path.